variants/DISCO_F413ZH/variant.cpp

```diff
@@ -0,0 +1,152 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2019, STMicroelectronics
+ * All rights reserved.
+ *
+ * This software component is licensed by ST under BSD 3-Clause license,
+ * the "License"; You may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at:
+ *                        opensource.org/licenses/BSD-3-Clause
+ *
+ *******************************************************************************
+ */
+
+#include "pins_arduino.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// Digital PinName array
+// This array allows to wrap Arduino pin number(Dx or x)
+// to STM32 PinName (PX_n)
+const PinName digitalPin[] = {
+  PF_6,  // D0 - UART7 RX
+  PF_7,  // D1 - UART7 TX
+  PG_13, // D2
+  PF_10, // D3
+  PB_6,  // D4
+  PE_6,  // D5
+  PB_0,  // D6
+  PC_13, // D7
+  PA_4,  // D8
```

Comment:
CubeMX pin planner says that PA4 is universal pin - GPIO, SPI and so on.

Reply:
This is specific to Arduino, CubeMx is not aware of how it is waorking in Arduino.
Yes PA4 can be used as a standard Digital pin, but because it also have Analog capability, Arduino identify it as analog pin (but still usable also in Arduino as digital).

Reply:
I'm stuck. How should I resolve this?

Reply:
I have done this.